release/release_tests.yaml

```diff
@@ -2599,45 +2599,45 @@
       script: python stress_tests/test_dead_actors.py --num-nodes=4 --num-parents=3
         --num-children=3
 
-- name: threaded_actors_stress_test
-  group: core-daily-test
-  working_dir: nightly_tests
-  legacy:
-    test_name: threaded_actors_stress_test
-    test_suite: nightly_tests
-
-  frequency: nightly
-  team: core
-  cluster:
-    cluster_env: stress_tests/stress_tests_app_config.yaml
-    cluster_compute: stress_tests/stress_test_threaded_actor_compute.yaml
-
-  run:
-    timeout: 7200
-    script: python stress_tests/test_threaded_actors.py --test-runtime 3600 --kill-interval_s
-      60
-
-    wait_for_nodes:
-      num_nodes: 201
-      timeout: 600
-
-    type: sdk_command
-    file_manager: sdk
-
-  smoke_test:
-    frequency: nightly
-    cluster:
```

Comment:
Can you just use the smoke test config for a regular test instead of completely disabling it? I think in this case stable: false can be removed

Reply:
How do I do that?

Reply:
And does the smoke test pass consistently?

Reply:
I think so! You can just 

```
- name: threaded_actors_stress_test
  group: core-daily-test
  working_dir: nightly_tests
  legacy:
    test_name: threaded_actors_stress_test
    test_suite: nightly_tests

  frequency: nightly
  team: core
  cluster:
    app_config: stress_tests/stress_tests_app_config.yaml
    compute_template: stress_tests/smoke_test_compute.yaml
  run:
    timeout: 3600
    script: python stress_tests/test_threaded_actors.py --test-runtime 1800 --kill-interval_s
        30

    wait_for_nodes:
      num_nodes: 5
      timeout: 600
```
(basically overwriting existing cluster env + run to the smoke test ones)